package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "restbase",
-  "version": "0.17.0",
+  "version": "0.18.0",
```

Comment:
It's not time to release the new version yet. We need to switch everything over before that can happen.

Reply:
I that case maybe let's not add the changelog now but just fix the example config and postpone adding the changelog until we actually make the release?

Reply:
Sure, +1